tests/Functional/BlobTest.php

```diff
@@ -46,6 +46,29 @@ public function testInsert(): void
         self::assertEquals(1, $ret);
     }
 
+    public function testInsertNull(): void
+    {
+        if (TestUtil::isDriverOneOf('ibm_db2')) {
+            self::markTestIncomplete('The ibm_db2 driver requires additional rework for this test to pass');
```

Comment:
https://github.com/doctrine/dbal/pull/5467 should be fixed in order to unblock fixing this issue.